docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -13,9 +13,22 @@
    "source": [
     "The purpose of the $T_2$ Hahn Echo experiment is to determine $T_2$ qubit property. \n",
     "\n",
-    "In this experiment, we would like to get a more precise estimate of the qubit's decay time. $T_2$ represents the amount of time required for the transverse magnetization to fall to approximately 37% ($\\frac{1}{e}$) of its initial value.\n",
+    "In this experiment, we would like to get a more precise estimate of the qubit's decay time. $T_2$ represents the amount of time required for a single qubit Bloch vector projection on the XY plane, to fall to approximately 37% ($\\frac{1}{e}$) of its initial amplitude.<br>\n",
+    "In <a href=\"./t2ramsey_characterization.ipynb\">Ramsey Experiment</a> we were introduced to the term <I>detuning frequency</I> (The difference between the frequency used for the control rotation, and the precise frequency).\n",
+    "<br>Hahn Echo experiment and CPMG sequence are experiments to estimate $T_2$ which are robust to the <I>detuning frequency</I>.\n",
+    "The decay in amplitude causes the probability function to take the following form:<br>\n",
+    "$$f(t) = A \\cdot e^{-\\frac{t}{T_2}}+ B$$\n",
+    "The difference between Hahn Echo and CPMG sequence is that in Hahn Echo experiment, there is only one echo sequences while in CPMG there are multiple echo sequences."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Decoherence Time\n",
+    "Decoherence time is the time taken for off-diagonal components of the density matrix to fall to approximately 37% ($\\frac{1}{e}$). For $t\\gg T_2$, the qubit statistics behave like a random bit. It gets the value of `0` with probability of $p$ and the value of `1` with probability of $1-p$.\n",
     "\n",
-    "Since the qubit is exposed to other types of noise (like $T_1$), we are using a $Rx(\\pi)$ pulse for decoupling and to solve our inaccuracy for the qubit frequency estimation."
+    "Since the qubit is exposed to other types of noise (like <a href=\"./t1.ipynb\"> T<sub>1</sub></a>), we are using a $Rx(\\pi)$ pulses for decoupling and to solve our inaccuracy for the qubit frequency estimation."
```

Comment:
```suggestion
    "Since the qubit is exposed to other types of noise (like <a href=\"./t1.ipynb\"> T<sub>1</sub></a>), we are using $Rx(\\pi)$ pulses for decoupling and to solve our inaccuracy for the qubit frequency estimation."
```

Reply:
Changed as suggested.